plugins/modules/network/routeros/routeros_api.py

```diff
@@ -0,0 +1,489 @@
+         selected path.
+        Example "ip address".
+         Eqvivalent of RouterOS cli "/ip address print"
+    required: true
+    type: str
+  add:
+    description:
+      - Will add selected arguments in selected path to RouterOS config.
+        Example "address=1.1.1.1/32 interface=ether1".
+         Equivalent in RouterOS cli
+         "/ip address add address=1.1.1.1/32 interface=ether1"
+    type: str
+  remove:
+    description:
+      - Remove config/value from RouterOS by '.id'.
+         Example "*03" will remove config/value with "id=*03"
+         in selected path.
+          Equivalent in RouterOS cli "/ip address remove numbers=1"
+         ,note "number" in RouterOS cli is different from ".id"
+    type: str
+  update:
+    description:
+      - Update config/value in RouterOS by ".id" in selected path.
+         Example ".id=*03 address=1.1.1.3/32" and path "ip address"
+         will replace existing ip address with ".id=*03".
+         Equivalent in RouterOS cli
+         "/ip address set address=1.1.1.3/32 numbers=1"
+         ,note number in RouterOS cli is different from ".id"
+    type: str
+  query:
+    description:
+      - Query given path for selected query attributes from
+         RouterOS aip and return '.id'.
+          WHERE is key word which extend query. WHERE format is
+         key operator value - with spaces.
+          WHERE valid operators are "==", "!=", ">", "<".
+          Example path "ip address" and query ".id address" will return
+         only ".id" and "address" for all items in "ip address" path.
+          Example path "ip address" and
+          query ".id address WHERE address == 1.1.1.3/32"
+         will return only ".id" and "address" for items in "ip address"
+          path, where address is eq to 1.1.1.3/32.
+          Example path "interface" and query "mtu name WHERE mut > 1400" will
+         return only interfaces "mtu,name" where mtu is bigger than 1400.
+          Equivalent in RouterOS cli "/interface print where mtu > 1400"
+    type: str
+  cmd:
+    description:
+      - Execute any/arbitrary command in selected path,
+         after the command we can add ".id".
+         Example path "system script" and cmd "run .id=*03"
+         is equivalent in RouterOS cli "/system script run number=0",
+          example path "ip address" and cmd "print"
+         equivalent in RouterOS cli "/ip address print"
+    type: str
+'''
+
+EXAMPLES = '''
+---
+- name: routeros_api test
+  hosts: localhost
+  gather_facts: no
+  vars:
+    hostname: "ros_api_hostname/ip"
+    username: "admin"
+    password: "secret_password"
+
+    path: "ip address"
+
+    nic: "ether2"
+    ip1: "1.1.1.1/32"
+    ip2: "2.2.2.2/32"
+    ip3: "3.3.3.3/32"
+
+    addips:
+      - "address={{ ip1 }} interface={{ nic }}"
+      - "address={{ ip2 }} interface={{ nic }}"
+
+    rmips:
+      - "{{ ip2 }}"
+      - "{{ ip3 }}"
+  tasks:
+    - name: get "{{ path }} print"
```

Comment:
```suggestion
    - name: Get "{{ path }} print"
```
etc.
This is a doc requirement.